server/sonar-server/src/main/java/org/sonar/server/platform/ServerSettingsImpl.java

```diff
@@ -35,17 +35,15 @@
  * <li>environment variables</li>
  * <li>system properties</li>
  * </ol>
- *
- * @since 2.12
  */
-public class WebServerSettings extends Settings implements ServerSettings {
+public class ServerSettingsImpl extends Settings implements ServerSettings {
```

Comment:
@simonbrandhof I don't understand this change

This class is actually specific to the WebServer (CE has it's own implementation called `ComputeEngineSettings`) and is not even in a package specific to the Web Server (such as `org.sonar.platform.web` or something)